ui/app/AppLayouts/Communities/views/EditAirdropView.qml

```diff
@@ -209,8 +215,7 @@ StatusScrollView {
             airdropTokens.forEach(entry => {
                 feesModel.append({
                     contractUniqueKey: entry.contractUniqueKey,
-                    title: qsTr("Airdropping %1 %2 on %3")
-                                     .arg(entry.amount * addresses.count)
+                    title: qsTr("Airdrop %1 on %3")
```

Comment:
```suggestion
                    title: qsTr("Airdrop %1 on %2")
```
This won't work at all otherwise

Reply:
Overlooked, bc there were 3 args before... Obvious bug, but surprisingly it was working fine :rofl: And missed bc of that.